e2e_testing/torchscript/basic.py

```diff
@@ -963,3 +963,38 @@ def forward(self, lhs):
 def TModuleRank0_basic(module, tu: TestUtils):
     module.forward(torch.tensor(7, dtype=torch.float32))
 
+class TensorLiteralModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+        torch.manual_seed(0)
+        self.t = torch.randn(2, 3)
```

Comment:
I think this should be `randint`. `randn` returns floats. Same for the second test